### ISSUE TITLE
Food recall warning: Keyboard focus is not visible for content under "Additional information"

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#111]
URL: https://hcdev.openplus.ca/en/alert-recall/certain-happy-veggie-world-hearts-choices-and-vegefarm-brand-vegan-and-vegetarian-1
User Agent: Firefox (v92), Chrome (v94), MS Edge
Language: English and French

Issue Description: When tabbing through, under “Additional information”, keyboard focus indicator is not visible for collapsed/expand content.

![image](https://user-images.githubusercontent.com/12256067/137019820-69e7f472-8117-41a9-9552-411b285d9c72.png)

Steps to reproduce:

1. Open the URL provided above
2. Navigate the page using the "Tab" key

Expected results: The keyboard focus indicator should be visible for those elements.

**Comment by echouin** [comment]
Sent to OP

**Comment by echouin** [comment]
From OP: ﻿done and will be part of next commit

**Comment by ndolter** [comment]
Validated in both languages in browsers: Chrome, Firefox and Edge, that items under “Additional information” now receive  keyboard focus when tabbing through.
